how-to/security_hardening.rst

```diff
@@ -0,0 +1,68 @@
+JAAS: Security Hardening
+========================
+JIMM, the service at the centre of JAAS can be hardened in a number of ways. This 
```

Comment:
I think you should mention the security overview doc somewhere to give the reader more context on where we stand security-wise